specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -13,7 +13,7 @@ events that require immediate user/developer action.
 * [The Windows Push Notification Services (WNS) Overview](https://docs.microsoft.com/windows/uwp/design/shell/tiles-and-notifications/windows-push-notification-services--wns--overview) Defines how an app today can subscribe to Push Notifications and send a payload down to the corresponding client.
 
 The below diagram is an illustration of the overall Push Flow that we have today:<br/>
-![Legacy Flow For Project Reunion](Legacy.png)
+![Legacy Flow For Project Reunion](Legacy.PNG)
```

Comment:
Thanks for fixing this. It's interesting that the linked files are case-sensitive!